Dockerfiles/cluster-agent/README.md

```diff
@@ -1,16 +1,21 @@
-# Cluster Agent 6 docker image
+# Datadog Cluster Agent | Containerized environments 
 
 This is how the official Datadog Cluster Agent (also known as `DCA`) image, available [here](https://hub.docker.com/r/datadog/cluster-agent/), is built.
 
 ## How to run it
 
 The following environment variables are supported:
 
-- `DD_API_KEY`: your API key (**required**)
+- `DD_API_KEY` - **required** - your [Datadog API key][https://app.datadoghq.com/account/settings#api].
 - `DD_HOSTNAME`: hostname to use for the DCA.
-- `DD_CLUSTER_AGENT_CMD_PORT`: Port you want the DCA to serve
+- `DD_CLUSTER_AGENT_CMD_PORT`: Port for the DCA to serve, default is `5005`.
+- `DD_USE_METADATA_MAPPER`: Enables the cluster level metadata mapping, default is `true`.
+- `DD_COLLECT_KUBERNETES_EVENTS` - configures the agent to collect Kubernetes events. See the [Event collection section](#event-collection) for more details.
+- `DD_LEADER_ELECTION`: activates the [leader election](#leader-election). You must set `DD_COLLECT_KUBERNETES_EVENTS` to `true` to activate this feature. Default value is `false`.
+- `DD_LEADER_LEASE_DURATION`: used only if the leader election is activated. See the details [here](#leader-election-lease). The expected value is a number of seconds.
```

Comment:
do we have a sane default value? If so we should say it here, if not we should set one *and* say it here.

Reply:
yes, 60 seconds https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L204